tests/Gedmo/Translator/Fixture/Person.php

```diff
@@ -12,54 +12,78 @@
 namespace Gedmo\Tests\Translator\Fixture;
 
 use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\DBAL\Types\Types;
 use Doctrine\ORM\Mapping as ORM;
 
 /**
  * @ORM\Entity
  */
+#[ORM\Entity]
 class Person
 {
     /**
+     * @var string|null
+     *
      * @ORM\Column(name="name", type="string", length=128)
      */
+    #[ORM\Column(name: 'name', type: Types::STRING, length: 128)]
     public $name;
 
     /**
+     * @var string|null
+     *
      * @ORM\Column(name="desc", type="string", length=128)
      */
+    #[ORM\Column(name: 'desc', type: Types::STRING, length: 128)]
     public $description;
 
     /**
+     * @var string|null
+     *
      * @ORM\Column(name="last_name", type="string", length=128, nullable=true)
      */
+    #[ORM\Column(name: 'last_name', type: Types::STRING, length: 128, nullable: true)]
     public $lastName;
+
     /**
+     * @var int|null
+     *
      * @ORM\Id
      * @ORM\GeneratedValue
      * @ORM\Column(type="integer")
      */
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: Types::INTEGER)]
     private $id;
 
     //
```

Comment:
We could remove these inline comments or move them to the property block.

Reply:
👍 I've removed them